ui/.neutrinorc.js

```diff
@@ -60,20 +60,20 @@ module.exports = {
         },
         proxy: {
           '/login': {
-            target: 'http://localhost:3050',
+            target: 'https://community-tc.services.mozilla.com/',
             changeOrigin: true,
           },
           '/graphql': {
-            target: 'http://localhost:3050',
+            target: 'https://community-tc.services.mozilla.com/',
             changeOrigin: true,
           },
           '/subscription': {
             ws: true,
             changeOrigin: true,
-            target: 'ws://localhost:3050',
+            target: 'https://community-tc.services.mozilla.com/',
           },
           '/api/web-server': {
-            target: 'http://localhost:3050',
+            target: 'https://community-tc.services.mozilla.com/',
             changeOrigin: true,
           },
```

Comment:
I suspect these changes were made during development?  You'll want to be careful not to check such changes in.  But, these changes shouldn't be needed if you follow the [Hacking the UI](https://github.com/taskcluster/taskcluster/blob/main/dev-docs/development-process.md#hacking-on-the-ui) steps.